### DISCUSSION TITLE
Consul or Euraka how to add request prefix

### DISCUSSION THREAD

**Discussion by zxuanhong** [prometheus/prometheus#12371] [Announcements]
## Consul or Euraka how to add request prefix
1. consul configured that the ng reverse proxy has a request prefix. How do I configure it
2. euraka has configured the content path. How to configure it

**Comment by roidelapluie** [comment]
For consul, let's have it in 2.45: https://github.com/prometheus/prometheus/pull/12372

**Comment by roidelapluie** [comment]
For eureka, you can use `server:8080/prefix` as server.
